rebench/configurator.py

```diff
@@ -225,6 +225,22 @@ def __init__(self, raw_config, data_store, ui, cli_options=None, cli_reporter=No
         experiments = raw_config.get('experiments', {})
         self._experiments = self._compile_experiments(experiments)
 
+    def _parse_executor(self, executor_name, executor_config):
+        path = executor_config.get('path')
+        executable = executor_config.get('executable')
+        cores = executor_config.get('cores')
+        version_command = executor_config.get('version_command')
+
+        executor = Executor([], False, self.ui, config_dir=self.config_dir)
+        if version_command:
+            executor.set_version_command(version_command)
+        return executor
+
+    def _compile_experiment(self, exp_name, experiment):
+        experiment['executors'] = {name: self._parse_executor(name, config) for name, config in
+                                   experiment.get('executors', {}).items()}
+        return Experiment.compile(exp_name, experiment, self)
+
```

Comment:
Could you elaborate a bit what this code is doing?

Isn't this what `model.executor.Executor.compile()` is doing?